src/gt4py/backend/dawn_backends.py

```diff
@@ -86,6 +103,20 @@ def _make_global_variables(self, parameters: list, externals: dict):
 
         return global_variables
 
+    def visit_Operator(self, op: enum.Enum):
```

Comment:
This is not a real `visit_` method to dispatch to Operator node, so please rename it to something like `make_operator` to make clear that this is a helper method.